<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtensionDialog</class>
 <widget class="QDialog" name="ExtensionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="primaryGroupBox">
         <property name="title">
          <string>&quot;&amp;Primary Key&quot;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="priColumnLabel">
            <property name="text">
             <string>Column:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="primaryColumnCombo">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="priOrderLabel">
            <property name="text">
             <string>Order:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="primaryOrderCombo">
            <item>
             <property name="text">
              <string>Ascendind</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Descending</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="secondaryGroupBox">
         <property name="title">
          <string>&quot;&amp;Secondary Key&quot;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="secColumnLabel">
            <property name="text">
             <string>Column:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="secondaryColumnCombo">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="secOrderLabel">
            <property name="text">
             <string>Order:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="secondaryOrderCombo">
            <item>
             <property name="text">
              <string>Ascendind</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Descending</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="tertiaryGroupBox">
         <property name="title">
          <string>&quot;&amp;Tertiary Key&quot;</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="terColumnLabel">
              <property name="text">
               <string>Column:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="tertiaryColumnCombo">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="triOrderLabel">
              <property name="text">
               <string>Order:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QComboBox" name="tertiaryOrderCombo">
              <item>
               <property name="text">
                <string>Ascendind</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Descending</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="moreButton">
         <property name="text">
          <string>&quot;&amp;More&quot;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>primaryColumnCombo</tabstop>
  <tabstop>primaryOrderCombo</tabstop>
  <tabstop>secondaryColumnCombo</tabstop>
  <tabstop>secondaryOrderCombo</tabstop>
  <tabstop>tertiaryColumnCombo</tabstop>
  <tabstop>tertiaryOrderCombo</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>moreButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ExtensionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ExtensionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>secondaryGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>tertiaryGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
